src/shogun-core-webapp-archetype/src/main/resources/archetype-resources/src/main/java/web/ProjectApplicationController.java

```diff
@@ -10,13 +10,13 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
 
-import ${package}.model.ProjectApplication;
-import ${package}.dao.ProjectApplicationDao;
+{package}.model.ProjectApplication;
+{package}.dao.ProjectApplicationDao;
```

Comment:
I think the `import` keyword got lost?

Reply:
Nice catch! Resolved in 2ecc1766aa4c2cf9afcd.